janusgraph-dist/src/assembly/static/conf/gremlin-server/janusgraph-berkeleyje-es-server.properties

```diff
@@ -1,26 +0,0 @@
-# Copyright 2019 JanusGraph Authors
```

Comment:
Identical to `janusgraph-berkeleyje.properties`. Thus, removing this file.

Reply:
I think you mean `conf/janusgraph-berkeleyje-es.properties`. One small difference I see between the two files is that the other doesn't contain the `index.search.hostname`. But I'm not sure if that is really helpful.
In general, I'm wondering why we had these `.properties` files under `gremlin-server` any way. Aren't we always using `.properties` files directly from the conf dir?

Reply:
You are right, I confused `conf/janusgraph-berkeleyje-es.properties` with `conf/janusgraph-berkeleyje.properties`. 

> One small difference I see between the two files is that the other doesn't contain the index.search.hostname 

That's was fixed in the previous commit. If you check these files in `master` branch, they are using identical properties but for `index.search.hostname` we use `127.0.0.1` instead of `elasticsearch` because in many cases `elasticsearch` isn't a default hostname for elasticsearch but for `127.0.0.1` it's clear that it expects local ES installation:
https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-dist/src/assembly/cfilter/conf/janusgraph-berkeleyje-es.properties
https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-dist/src/assembly/static/conf/gremlin-server/janusgraph-berkeleyje-es-server.properties
`index.search.hostname` isn't necessary to be used because it already default to `127.0.0.1` but I was thinking about it more like about documentation. I noticed that this property was used in `conf/gremlin-server/janusgraph-berkeleyje-es-server.properties` and thus decided to add it in the previous commit.

>  In general, I'm wondering why we had these .properties files under gremlin-server any way. Aren't we always using .properties files directly from the conf dir?

It's a good question but I wonder it myself. I can only guess that these files were initially used for `conf/gremlin-server/*.yaml` configurations but than become obsolete after some time and nobody removed them but I could be wrong. I couldn't find other usages of these files.